README.md

```diff
@@ -928,6 +928,23 @@ Other Style Guides
     );
     ```
 
+  <a name="functions--prefer-rest-params"></a><a name="7.16"></a>
```

Comment:
we don't add new sections with numbered links; those are legacy.

Reply:
I tried to be consistent with the above points as they had been written. If still their is something wrong then can you help me how can I fixed that.

Reply:
There’s no consistency here - just that anything added in the last couple years will not have a numbered link (`<a name`), and all the older sections will. New ones must not have it.

Reply:
their is any refence from where I can take a look. Then I will update this.

Reply:
There are many other sections that are newly added; off the top of my head i can’t recall any specifically.

Reply:
done.